spec/models/web_hook_spec.rb

```diff
@@ -64,6 +64,17 @@
 
         WebHook.enqueue_hooks(:post, post_id: 1)
       end
+
+      it 'registers related event on DiscourseEvent' do
```

Comment:
They can go under this file. :) For example, creating a post should give us a webhook or at least make sure the job is enqueued if we have specs covering the job